src/main/java/com/conveyal/r5/analyst/cluster/PathWriter.java

```diff
@@ -33,22 +32,26 @@
 
     OutputStream outputStream;
 
-    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList) {
+    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList, int destinations, int iterations) {
```

Comment:
I would call these `nDestinations` and `nIterations` to emphasize that the variables don't contain the destinations and iterations themselves, but are just counts/sizes.